winrm/winrm.go

```diff
@@ -211,9 +211,13 @@ func NewClient(config ClientConfig) (*Client, error) {
 // If the Run successfully executes it returns nil
 func (c *Client) Run(command string, stdout io.Writer, stderr io.Writer) error {
 	logger.Debugf("Runing cmd on WinRM connection %q", command)
-	_, err := c.conn.Run(command, stdout, stderr)
-	if err != nil {
-		return errors.Annotatef(err, "cannot run WinRM command %q", command)
+	exitCode, err := c.conn.Run(command, stdout, stderr)
+	if exitCode != 0 {
+		var cmdErr error = err
```

Comment:
Why create `cmdErr` here? Doesn't seem like you really need it.

Reply:
Isolation is getting to me.